DB.sql

```diff
@@ -0,0 +1,41 @@
+DROP DATABASE IF EXISTS ttb;
+
+CREATE DATABASE ttb;
+
+USE ttb;
+
+
+CREATE TABLE civilian (
+	civilian_id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    age INT NOT NULL,
+    ethnicity VARCHAR(20) NOT NULL, -- maybe some consistency issue here?
+    gender ENUM('male', 'female', 'other', 'prefer not to say') NOT NULL
```

Comment:
As we mentioned, [enums](http://komlenic.com/244/8-reasons-why-mysqls-enum-data-type-is-evil) are bad because they are expensive and inconvenient to change.